src/ImageGallery.react.jsx

```diff
@@ -351,6 +352,7 @@ export default class ImageGallery extends React.Component {
                       onClick={this._wrapClick(swipeNext)}/>
                   ],
                 <Swipeable
+                  className='image-gallery-swipe'
```

Comment:
I'm curious why you need a class here? You can select the container from `image-gallery-slides`